lib/puppet-languageserver/message_router.rb

```diff
@@ -60,6 +60,45 @@ def receive_request(request)
         PuppetLanguageServer.log_message(:debug, 'Received shutdown method')
         request.reply_result(nil)
 
+      when 'pdk/newClass'
+        begin
+          PuppetLanguageServer::Handlers::PdkHandler.newClass(
+            request.params['name'],
+            request.params['targetdir']
+          )
+          request.reply_result(LSP::PdkResponse.new('data' => request.params['name']))
```

Comment:
@glennsarti I'd like to return an array of files created (in this case its the class and spec files), instead of just a single string. How do I create a LSP class to do this?